nrf52-hal-common/Cargo.toml

```diff
@@ -1,41 +1,48 @@
 [package]
+name = "nrf52-hal-common"
+version = "0.6.0"
+description = "Common HAL for the nRF52 family of microcontrollers.  More specific HAL crates also exist."
+
+repository = "https://github.com/nrf-rs/nrf52-hal"
 authors = [
-    "James Munns <james.munns@gmail.com>",
-    "Hanno Braun <hanno@braun-robotics.com",
-    "Wez Furlong <wez@wezfurlong.org",
+    "James Munns <james@onevariable.com>",
+    "Hanno Braun <hanno@braun-robotics.com>",
+    "John Scarrott <johnps@outlook.com>",
+    "Wez Furlong <wez@wezfurlong.org>",
 ]
+
 categories = ["embedded", "hardware-support", "no-std"]
-description = "Common HAL for the nRF52 family of microcontrollers.  More specific HAL crates also exist."
 keywords = ["arm", "cortex-m", "nrf52", "hal"]
 license = "MIT OR Apache-2.0"
-name = "nrf52-hal-common"
-repository = "https://github.com/nrf-rs/nrf52-hal"
-version = "0.0.1"
+edition = "2018"
 
 [dependencies]
-cortex-m = "0.5.4"
+cortex-m = "0.5.8"
 nb = "0.1.1"
-void = { version = "1", default-features = false }
+
+[dependencies.void]
+default-features = false
+version = "1.0.2"
 
 [dependencies.cast]
 default-features = false
 version = "0.2.2"
 
-[dependencies.embedded-hal]
-features = ["unproven"]
-version = "0.2.1"
-
-[dependencies.nrf52]
+[dependencies.nrf52832-pac]
 optional = true
-version = "~0.4"
+version = "=0.6.0"
```

Comment:
Why the strict requirement? Seems like that will cause trouble down the line sooner or later (when we release a new `nrf52832-pac` and forget to update the dependency here).

Reply:
Good call! Removed (and all following)